lib/config.js

```diff
@@ -137,17 +139,19 @@ function getLocalConfig(thisConfig, directory) {
             continue;
         }
 
+        found = true;
+        debug(`Using ${localConfigFile}`);
+        config = ConfigOps.merge(localConfig, config);
+
         // Check for root flag
         if (localConfig.root === true) {
             rootPath = path.dirname(localConfigFile);
+            thisConfig.rootFound = true;
+            thisConfig.rootConfig = config;
+            break;
         }
-
-        found = true;
-        debug(`Using ${localConfigFile}`);
-        config = ConfigOps.merge(localConfig, config);
     }
-
-    if (!found && !thisConfig.useSpecificConfig) {
+    if (!found && !thisConfig.useSpecificConfig && !thisConfig.rootFound) {
```

Comment:
I think this is redundant because `found` will always be true if `rootFound` is true.